### PR TITLE
Update flysysteme service

### PR DESCRIPTION
**Pull request by ottaviano**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -135,6 +134,7 @@
         "fabpot/goutte": "^3.2",
         "friendsofphp/php-cs-fixer": "^2.16",
         "fzaninotto/faker": "^1.9",
+        "league/flysystem-memory": "^1.0",
```
Comment (kevin-verschaeve) [inline composer.json:137]: why did you moved this ine require-dev part ? It seems used in prod env too

Comment (ottaviano) [reply composer.json:137]: @kevin-verschaeve, it seems to be used only on test, no?
image

Comment (kevin-verschaeve) [reply composer.json:137]: my bad, misread the package name !

**Review by kevin-verschaeve — APPROVED** [review]

**Comment by ottaviano** [comment]
[ch12734]

**Pull request merged**